editor.planx.uk/src/ui/editor/SortFlowsSelect.tsx

```diff
@@ -0,0 +1,152 @@
+import TrendingDownIcon from "@mui/icons-material/TrendingDown";
+import TrendingUpIcon from "@mui/icons-material/TrendingUp";
+import Box from "@mui/material/Box";
+import IconButton from "@mui/material/IconButton";
+import MenuItem from "@mui/material/MenuItem";
+import {
+  FlowSummary,
+  FlowSummaryOperations,
+  PublishedFlowSummary,
+} from "pages/FlowEditor/lib/store/editor";
+import React, { useEffect, useState } from "react";
+import { useNavigation } from "react-navi";
+
+import SelectInput from "./SelectInput/SelectInput";
+
+type SortDirection = "asc" | "desc";
+type SortKeys = keyof FlowSummary;
+type SortNestedKeys = keyof PublishedFlowSummary | keyof FlowSummaryOperations;
+type SortTypes = SortKeys | SortNestedKeys;
+
+interface BasicSort {
+  displayName: string;
+  sortKey: Exclude<SortKeys, "publishedFlows" | "operations">;
+}
+
+interface PublishedFlowSort {
+  displayName: string;
+  sortKey: "publishedFlows";
+  nestedKey: keyof PublishedFlowSummary;
+}
+
+type SortObject = PublishedFlowSort | BasicSort;
+
+const sortArray: SortObject[] = [
+  { displayName: "Name", sortKey: "name" },
+  { displayName: "Last updated", sortKey: "updatedAt" },
+  { displayName: "Status", sortKey: "status" },
```

Comment:
nit: in additional to all of Daf's comments, a small thing here is that I don't believe `status` should be an attribute included in the "sort", but rather "filter" in the future (eg because it has categorical values)